modules/preview-react/InformationHighlight/lib/parts/Heading.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {createComponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {createStyles, handleCsProp} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Heading, TypeLevelProps} from '@workday/canvas-kit-react/text';
+
+const headingStyles = createStyles({
+  ...system.type.body.small,
+  color: base.blackPepper400,
+  gridColumn: '2',
+  fontWeight: system.fontWeight.bold,
+  marginTop: system.space.zero,
+  marginBottom: system.space.zero,
+});
+
+export interface InformationHighlightHeadingProps
+  extends Omit<Partial<ExtractProps<typeof Heading, never>>, 'size'> {
+  size?: TypeLevelProps['size'];
+}
```

Comment:
```suggestion
export interface InformationHighlightHeadingProps
  extends Partial<ExtractProps<typeof Heading, never>> {}
```

Sorry it think this is cleaner, it just makes all props from heading optional.